src/main/kotlin/io/zerobase/smarttracing/GraphDao.kt

```diff
@@ -9,9 +9,12 @@ import io.zerobase.smarttracing.models.ScanId
 import io.zerobase.smarttracing.models.SiteId
 import io.zerobase.smarttracing.models.ScannableId
 import io.zerobase.smarttracing.models.OrganizationId
+import io.zerobase.smarttracing.models.UserId
 import org.neo4j.driver.Driver
 import java.util.*
 
+data class UserDaoStruct(val name: String?, val phone: String?, val email: String?, val id: String)
```

Comment:
putting `Struct` in the name of a class is an anti-pattern. just make a `User` in the models package and have the dao return it.